## Aufgabe 6.1
Ein Weinhändler behauptet, dass die von ihm gefüllten Weinflaschen 70 Zentiliter
enthalten. Ein skeptischer Konsument vermutet aber, dass der Weinhändler zu wenig
Wein abfüllt und will diese Behauptung überprüfen. Deshalb kauft er 12 Weinflaschen
und misst ihren Inhalt. Er findet:
71, 69, 67, 68, 73, 72, 71, 71, 68, 72, 69, 72 (in Zentiliter)
Nehmen Sie zunächst an, dass die Standardabweichung der Abfüllung im Voraus
bekannt ist. Sie beträgt σ = 1.5 Zentiliter. Da die Standardabweichung der Messungen bekannt ist, können wir einen z-Test durchführen. Führen Sie den (einseitigen; in
welche Richtung?) Test auf dem 5 %- Signifikanzniveau durch.
Geben Sie die Modellannahmen, H0, HA, den Verwerfungsbereich, den Wert der Teststatistik und das Testergebnis explizit an.
Formulieren Sie in einem Satz die Schlussfolgerung für den kritischen Konsumenten.

H0 = u = u0 = 70

In [1]:
from scipy.stats import norm
import numpy as np

ew = 70
n = 12
sa = 1.5
a = 0.05

norm.ppf(q=a, loc=ew, scale=sa/np.sqrt(n))

69.28775748677653

In [2]:
# Entscheidung
from pandas import Series
wein = Series([71, 69, 67, 68, 73, 72, 71, 71, 68, 72, 69, 72])
wein.mean()

70.25

0 Hypothese wird nicht verworfen da 70.25 nicht im Verwerfungsbereich liegt.

## Aufgabe 6.2
Ein Weinhändler behauptet, dass die von ihm gefüllten Weinflaschen 70 Zentiliter
enthalten. Ein skeptischer Konsument vermutet aber, dass der Weinhändler zu wenig
Wein abfüllt und will diese Behauptung überprüfen. Deshalb kauft er 12 Weinflaschen
und misst ihren Inhalt. Er findet:
71, 69, 67, 68, 73, 72, 71, 71, 68, 72, 69, 72 (in Zentiliter)
Die Standardabweichung der Abfüllungen ist nicht bekannt. Man muss sie also aus
den gemachten Stichproben schätzen:
σˆ
2
x =
1
n − 1
n
∑
i=1
(xi − x)
2 ≈ 1.962
Da die Standardabweichung geschätzt wurde und nicht mehr exakt bekannt ist, kann
der z-Test nicht durchgeführt werden. Verwenden Sie nun den t-Test auf dem 5 %-
Signifikanzniveau.
Geben Sie die Modellannahmen, H0, HA, den Verwerfungsbereich, den Wert der Teststatistik und das Testergebnis explizit an.
1
Was ändert sich an obigem Test?

In [4]:
sa = wein.std()
sa

1.9598237397554634

In [6]:
from scipy.stats import t
t.ppf(q=a, df=n-1, loc=ew, scale=sa/np.sqrt(n))

68.98397388627933

In [7]:
from pandas import Series
wein = Series([71, 69, 67, 68, 73, 72, 71, 71, 68, 72, 69, 72])
wein.mean()

70.25

## Aufgabe 6.3
Unterhalb einer Kläranlage wurden 16 unabhängige Wasserproben aus einem Fluss
entnommen und jeweils deren Ammoniumkonzentration Xi
(in µg NH_4-N/l) mit
einem Messgerät bestimmt. Der Mittelwert der Proben ergab x16 = 204.2.
Wir wollen nun wissen, ob mit diesem Experiment eine Überschreitung des Grenzwerts von 200 µg NH_4-N/l nachgewiesen werden kann (auf dem 5 % Niveau).

a) Nehmen Sie an, die Standardabweichung der Messungen sei im Voraus aufgrund früherer Studien bekannt. Sie betrage 10 µg NH_4-N/l.
Führen Sie unter dieser Annahme einen z-Test durch, um zu prüfen, ob eine
Grenzwertüberschreitung nachgewiesen werden kann.
Geben Sie die Modellannahmen, H0, HA, den Verwerfungsbereich, den Wert
der Teststatistik und das Testergebnis explizit an.

In [8]:
import numpy as np
from scipy.stats import norm

n = 16
ew = 200
sa = 10
a = 0.05

norm.ppf(q=1-a, loc=ew, scale=sa/np.sqrt(n))

204.1121340673787

Entscheidung: der Gemessene wert 204.2 liegt im Verwerfungsbereich, damit wird die 0 Hypothese Verworfen und die überschreitung von 200 µg NH_4-N/l konnte nachgewiesen werden.

b) Wie wahrscheinlich ist es, dass man mit 16 unabhängigen Wasserproben eine
Grenzwertüberschreitung nachweisen kann, wenn die wahre Ammoniumkonzentration tatsächlich über dem Grenzwert, und zwar bei 205 µg NH_4-N/l
liegt?

In [10]:
1-norm.cdf(x=204.112134, loc=205, scale=sa/np.sqrt(n))

0.6387600414072817

c) Wie wahrscheinlich ist es, dass man mit 16 unabhängigen Wasserproben fälschlicherweise eine Grenzwertüberschreitung nachweist, obwohl die wahre Ammoniumkonzentration bei 200 µg NH-4-N/l liegt und den Grenzwert somit genau einhält?

In [11]:
1-norm.cdf(x=204.112134, loc=200, scale=sa/np.sqrt(n))

0.05000000277965755

d) Nehmen Sie an, dass die Standardabweichung von 10 µg/l aus den 16 Proben
geschätzt worden ist. Deshalb ist nun ein t-Test (Nullhypothese µ0 = 200 µg/l)
und nicht ein z-Test angebracht. Führen Sie den t-Test durch.


In [12]:
import numpy as np
from scipy.stats import t

t.ppf(q=0.95, df=n-1, loc=200, scale=10/np.sqrt(n))

204.38262588923138

Mit einem T-Test liegt die Messung knapp ausserhalb des Verwerfungsbereichs. die 0 Hypothese wird nicht verworfen.